### ISSUE TITLE
0.22.3 crashed after packaging 

### ISSUE THREAD

**Issue by Arti3DPlayer** [jyapayne/Web2Executable#200]
When i migrated to 0.22.3 and built app with web2exe I got this:
https://github.com/nwjs/nw.js/issues/5884

Looks like `nw.js` changed file hierarchy

**Comment by Arti3DPlayer** [comment]
On windows is working, on Mac - doesn't work. On Linux didn't test

**Comment by eemi2010** [comment]
Actually it looks like any version above 22 crashes on mac

**Comment by eemi2010** [comment]
the latest functioning version for me is 0.19.5

**Comment by Blatman** [comment]
Yup, I tried 0.21.6 and was OK on OSX but the symlinks in 0.22 and later have killed this app deader than the proverbiable duck - have to try and get my head around nwjs-builder-phoenix now .... :-(
